LR_Tourist/DA/Data/HotelDTO.cs

```diff
@@ -0,0 +1,15 @@
+﻿
+namespace DA.Data
+{
+    public class HotelDTO : IEntityBase
+    {
+        public int Id { get; }
```

Comment:
In this case, Id also should have a setter.
Only in IEntityBase interface can be get only.

PS You can have only single IEntityBase in DAL that can be reused in BL.